### ISSUE TITLE
Make custom webhook host supports IP address as well

### ISSUE THREAD

**Issue by raymanound** [opendistro-for-elasticsearch/alerting#137]
Hi, I have a use case that sending request to localhost with data but the host in custom attributes URL does not accept either "localhost" or "127.0.0.1". Can you consider to make it accept those values? Thanks.
![image](https://user-images.githubusercontent.com/58698476/71560053-f19d3580-2a9f-11ea-910e-12604436d159.png)

**Comment by dbbaughe** [comment]
Hi @raymanound,

Were you able to get around this by using the API to create the destination?
Just curious if this is issue purely in the Kibana plugin or both.

**Comment by H1L021** [comment]
hello @dbbaughe 
I have this same limitation. I wanna send alerts to TheHive with custom webhooks but i can't use IPs addresses.
I defined the TheHive server IP and mapped it with a domain name in the /etc/hosts file. When I use the same command from the opendistro node to thehive node it works fine and create the alerts. However, when i defined it in the kibana alerting plugin and created an alert, I got this error message : 

```
"alert_history" : [
            {
              "timestamp" : 1589756179815,
              "message" : """Error running action:
java.net.UnknownHostException: thehive.hidden.org: Name or service not known"""
            }
          ], 
```
I was wondering what is the domain resolution mechanism the alerting plugin is using since it works with curl and not with it.
